azure-pipelines.yml

```diff
@@ -8,11 +8,11 @@ jobs:
           test_title: Pylint All
           toxenv: pylint
           python: 3.7
-        # Pylint-Subpackages:
-        #   image_name: ubuntu-16.04
-        #   test_title: Pylint Subpackages
-        #   toxenv: analysis,dicom,fileformats,labs,logfiles,coordsandscales,electronfactors,databases,utilities,workshops,toolbox,xlwings
-        #   python: 3.7
+        Test-Subpackages:
+          image_name: ubuntu-16.04
+          test_title: Test Subpackages
+          toxenv: analysis,dicom,fileformats,labs,logfiles,coordsandscales,electronfactors,databases,utilities,workshops,toolbox,xlwings
+          python: 3.7
```

Comment:
We really do need this... especially if I am refactoring packages. It would really suck to release standalone packages which won't work in a standalone environment ...